packages/fields/src/types/Content/views/Field.js

```diff
@@ -45,7 +21,7 @@ let ContentField = ({ field, value: serverValue, onChange, autoFocus }) => {
             <Editor
               blocks={field.getBlocks()}
               value={value}
-              onChange={setValue}
+              onChange={onChange}
```

Comment:
This has a perf impact because we're no longer wait for `onBlur` before triggering the whole state update cycle.

But, it also fixes a bug where clicking the "+" would blur the content editor, then the user would inject an image (for example), which changes the value of the field, but doesn't trigger another blur event, so now the local `value` is stale.